src/pac-file-tester.ts

```diff
@@ -92,7 +89,12 @@ const vmContext = (ip: string) => {
   }
 
   const isPlainHostName = (host: string) => {
-    return (host.search('\\\\.') == -1);
+    return host.indexOf(".") === -1;
+  }
+
+  const dnsDomainIs = (host: string, domain: string) => {
+    console.log(`${host} compare to ${domain}`);
```

Comment:
Added `dnsDomainIs`  as it was in a proxy file I was testing:

see https://findproxyforurl.com/pac-functions/